libraries/cyclestreets-fragments/src/main/java/net/cyclestreets/itinerary/SegmentAdapter.kt

```diff
@@ -18,7 +18,7 @@ import net.cyclestreets.util.Turn
 import net.cyclestreets.util.TurnIcons
 
 internal class SegmentAdapter(context: Context) : BaseAdapter() {
-    private val footprints: Drawable = ResourcesCompat.getDrawable(context.resources, R.drawable.footprints, null)!!
+    private val footprints: Drawable = ResourcesCompat.getDrawable(context.resources, R.drawable.footprints2, null)!!
```

Comment:
We need to talk about images (see https://github.com/cyclestreets/android/issues/515#issuecomment-1236974153).

Reply:
Might be better to keep this as a separate issue.